### DISCUSSION TITLE
Have an abbreviation expansion example?

### DISCUSSION THREAD

**Discussion by aetkie** [UNC-Project-Open-AAC/OS-DPI#226] [Q&A] [answered]
Working on some updates on the Terminology page of the wiki. Does anyone have an example of abbreviation expansion?

**Comment by gbishop** [comment]
Where does it get used in the other pages?

That term makes me think of the ways people shorten text messages.

BRB = be right back
BRT = be right there

That kind of thing.

**Comment by sofia-bg** [comment]
There might be a really old version where Lori was working on it… but I don’t remember where we landed on getting it to work. I think we moved on to working on predictive text.

From: Andrea Etkie ***@***.***>
Date: Tuesday, October 24, 2023 at 9:04 AM
To: UNC-Project-Open-AAC/OS-DPI ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [UNC-Project-Open-AAC/OS-DPI] Have an abbreviation expansion example? (Discussion #226)

Working on some updates on the Terminology page of the wiki. Does anyone have an example of abbreviation expansion?

**Reply by aetkie** [reply]
so do we think it is not a currently working function in OS-DPI?

**Comment by gbishop** [comment, marked as answer]
I can't think of any special functions for implementing it. One could be
added. I bet you could get clever with a keyboard design and make it happen
on the space key. It could work like prediction but instead of a list of
predictions that are displayed in a grid, you'd have many entries for the
Space key in the content. If the $prefix  is "brb" (say) then pressing the
space key would replace it with "be right back".

Ha! Yet another thing you can simulate.

I'm not sure you need an entry in the terminology section for this.

**Reply by sofia-bg** [reply]
@aetkie ^this is really what I meant, that we agreed it _can_ be done but that I don't know that we built it into any examples. If it is @lageist might have a working example

**Reply by aetkie** [reply]
Right. It can definitely be done. But has anyone incorporated it into a design yet? It appears not, so I removed the term from the list.

**Reply by lageist** [reply]
@aetkie and @sofia-bg I'll look around, but I don't recall actually building this, but the approach @gbishop notes above was the plan. Could be easily added to one of the keyboard with prediction designs.

**Reply by sofia-bg** [reply]
if we want to keep the term to demonstrate the potential seems easy enough to hack up...
